plugins/uefi-capsule/fu-uefi-tool.c

```diff
@@ -62,55 +65,121 @@ main (int argc, char *argv[])
-			_("Apply firmware updates"),
-			/* TRANSLATORS: command argument: uppercase, spaces->dashes */
-			C_("A single GUID", "GUID") },
-		{ "flags", 'f', 0, G_OPTION_ARG_STRING, &flags,
-			/* TRANSLATORS: command line option */
-			_("Use quirk flags when installing firmware"), NULL },
-		{ NULL}
-	};
+	    {"verbose",
+	     'v',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &verbose,
+	     /* TRANSLATORS: command line option */
+	     _("Show extra debugging information"),
+	     NULL},
+	    {"version",
+	     '\0',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_version,
+	     /* TRANSLATORS: command line option */
+	     _("Display version"),
+	     NULL},
+	    {"log",
+	     'L',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_log,
+	     /* TRANSLATORS: command line option */
+	     _("Show the debug log from the last attempted update"),
+	     NULL},
+	    {"list",
+	     'l',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_list,
+	     /* TRANSLATORS: command line option */
+	     _("List supported firmware updates"),
+	     NULL},
+	    {"supported",
+	     's',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_supported,
+	     /* TRANSLATORS: command line option */
+	     _("Query for firmware update support"),
+	     NULL},
+	    {"info",
+	     'i',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_info,
+	     /* TRANSLATORS: command line option */
+	     _("Show the information of firmware update status"),
+	     NULL},
+	    {"enable",
+	     'e',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_enable,
+	     /* TRANSLATORS: command line option */
+	     _("Enable firmware update support on supported systems"),
+	     NULL},
+	    {"esp-path",
+	     'p',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_STRING,
+	     &esp_path,
+	     /* TRANSLATORS: command line option */
+	     _("Override the default ESP path"),
+	     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+	     _("PATH")},
+	    {"set-debug",
+	     'd',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_set_debug,
+	     /* TRANSLATORS: command line option */
+	     _("Set the debugging flag during update"),
+	     NULL},
+	    {"unset-debug",
+	     'D',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_NONE,
+	     &action_unset_debug,
+	     /* TRANSLATORS: command line option */
+	     _("Unset the debugging flag during update"),
+	     NULL},
+	    {"apply",
+	     'a',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_STRING,
+	     &apply,
+	     /* TRANSLATORS: command line option */
+	     _("Apply firmware updates"),
+	     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+	     C_("A single GUID", "GUID")},
+	    {"type",
+	     't',
+	     G_OPTION_FLAG_NONE,
+	     G_OPTION_ARG_STRING,
+	     &type,
+	     /* TRANSLATORS: command line option */
+	     _("Set the device type"),
```

Comment:
Maybe when presenting this it should be "device update method"

Reply:
Fixed, thanks.